tests/e2eTestForBoost/step-definitions/boost.steps.js

```diff
@@ -1,39 +1,31 @@
 const {Given, When, Then} = require('@cucumber/cucumber')
 let helpers = require('../../../helpers/common-actions');
-const assert = require('assert');
 
 
-Given(/^Login with (.*) and (.*)$/, async (username, password) => {
-   await boostPostPage.performLogin(username, password)
+Given(/^I login with (.*) and (.*)$/, async (username, password) => {
+   await boostPostPage.login(username, password)
 })
-
-Then (/^Create a text post$/,async function (){
-    await myFeedPage.CreatePost()
-    await helpers.waitForTimeout(page,5000);
-    await  myFeedPage.CreatePostFunc();
+Then(/^I create a text post$/,async function (){
+    await myFeedPage.createPost();
+    await helpers.waitForTimeout(page,10000);
+    await myFeedPage.validateCreatedPost();
 })
-
-Then (/^I click the boost button$/,async function (){
+Then(/^I click the Boost button$/,async function (){
     await myFeedPage.clickBoost();
 })
-
-When (/^I check the Boost Campaign Details$/,async function (){
-    await boostPage.VerifySetBudgetAndDuration();
+When(/^I check the Boost Campaign details$/,async function (){
+    await boostPage.verifySetBudgetAndDuration();
     await boostPage.setBudgetAndDuration();
 })
-
-When (/^Complete the Boost payment process$/,async function (){
+When(/^I complete the Boost payment process$/,async function (){
     await helpers.waitForTimeout(page,5000);
-    await orderSummaryPage.VerifyOrderSummary();
-    await orderSummaryPage.CompletePurchase();
+    await orderSummaryPage.verifyOrderSummary();
+    await orderSummaryPage.completePurchase();
 })
-
 When(/^I click on the Campaign Dashboard button$/,async function (){
-    await helpers.waitForTimeout(page,10000);
     await campaignDashboardPage.campaignDashBoardButton();
 })
-
-Then (/^Boost Post Campaign is returned should be returned in the Campaign Dashboard$/,async function (){
+Then(/^Boost Post campaign is returned should be returned in the Campaign Dashboard$/,async function (){
```

Comment:
Is it possible to receive a Boost Post campaign from a parallel/previous test?
If yes, is there any way to ensure, that we are verifying boost of the specific post, created in this test by the earlier step?

Reply:
I was trying to find some x-path for this, but then it got super complicated, then I realised that the Boost locator will always click the first button which is always the post we created in Step 1 "Create a text post" , if that fails all tests will stop. So going with the simple logic for now.

Reply:
I mean, is it possible to fail on boosting a post, but still pass the `Boost Post campaign is returned should be returned in the Campaign Dashboard` because we have already a boosted post(for example, from the previous test)?

Reply:
Thank you for the review @stereohorse , yes to avoid this case to pass the "Boost Post campaign is returned should be returned in the Campaign Dashboard" step, What I did now is: 
1. Created a Text post with a Random test so everytime we have a unique post
2. In Campaign Dashboard step - it will check for that unique post & pass/fail accordingly 

Hope this makes sense :)